### PR TITLE
Add synmonitoring label to namespace

### PR DESCRIPTION
**Pull request by laserb**
It is creating a ServiceMonitor, but the label is missing.

## Checklist

- [x] Keep pull requests small so they can be easily reviewed.

### CODE REVIEW

**Comment by laserb** [comment]
Draft because it probably is missing a network policy that allows monitoring, too.

**Comment by laserb** [comment]
Added missing network policy to allow monitoring from syn prometheus.

**Review by simu — APPROVED** [review]
LGTM. Note that this PR only addresses monitoring for Keycloak on clusters which use component `rancher-monitoring`.

File: component/prometheus-netpol.jsonnet [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        matchLabels: {
+          'app.kubernetes.io/instance': keycloak_name,
+          'app.kubernetes.io/name': keycloak_name,
+        },
+      },
+    },
+  };
+
+{
+  '20_netpol': netpol,
```
Comment (ccremer) [inline component/prometheus-netpol.jsonnet:58]: number 20 is already taken by LE certificate
```suggestion
  '40_netpol': netpol,
```

**Review by ccremer — APPROVED** [review]

**Pull request merged**